OpenSSL/crypto.py

```diff
@@ -1472,8 +1472,34 @@ def get_extension(self, index):
 
 class X509Store(object):
     """
-    An X509 certificate store.
-    """
+    An X.509 store.
+
+    An X.509 store is used to describe a context in which to verify a
+    certificate. A description of a context may include a set of certificates
+    to trust, a set of certificate revocation lists, verification flags and
+    more.
+
+    An X.509 store, being only a description, cannot be used by itself to verify
+    a certificate. To carry out the actual verification process, see
+    :py:class:`X509StoreContext`.
+    """
+
```

Comment:
Not all of the flags below are *tested*. What do folks prefer I do here: 1) expose all of the flags below and let people use them even though they're not tested or 2) only expose the flags that have been specifically tested?